app.js

```diff
@@ -13,10 +13,20 @@ app.get('/api/articles/:article_id', getArticlesById);
 
 app.get('/api/articles', getAllArticles)
 
+app.get('/api/articles/:article_id/comments', getAllCommentsOnArticle)
+
 app.all('*', (req, res, next) => {
     res.status(404).send({msg: 'path not found'})
 });
 
+app.use((err, req, res, next) => {
+    if(err.code === '22P02'){
+        res.status(400).send({msg: 'Article ID submitted is invalid datatype'})
```

Comment:
You might find that this error code appears in other circumstances, think about reducing the specificity of this error message if you need to.